crates/compiler/fmt/src/test_helpers.rs

```diff
@@ -0,0 +1,80 @@
+use bumpalo::Bump;
+use roc_test_utils::assert_multiline_str_eq;
+
+use crate::{
+    annotation::{Formattable, Newlines, Parens},
+    Buf,
+};
+
+/// Check that the given input is formatted to the given output.
+/// If the expected output is `None`, then this only checks that the input
```

Comment:
It looks like `check_formatting` doesn't return None - is this comment outdated?